templates/home.py

```diff
@@ -0,0 +1,19 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from Models.Reviews import Review
+URI = "postgres://xeeqkeipvcflil:fead0f891152e8f657d7ee59ba256aa36c24fed629a0ee765b6a6aac6da2610f@ec2-54-225-236-102.compute-1.amazonaws.com:5432/d7mipabothqsm0"
```

Comment:
Traditionally, we stored it in a config file that was `.gitignore`d and that we manually passed distributed around (although this is admittedly a bit of a PITA). The other way to do it is to have a `dev` environment where Postgres lives on the local machine (somewhere on `localhost`) and worry about a `prod` environment later.

There are lots of different ways to tackle this problem -- you should ask your mentor about what they think is best.